package.json

```diff
@@ -46,7 +51,8 @@
   },
   "browserify": {
     "transform": [
-      "glslify"
+      "glslify",
+      "cssify"
```

Comment:
Has this been rebased against develop?

There are regressions in this package json

For example the removal of the tape script